front/lib/api/assistant/conversation.ts

```diff
@@ -595,19 +594,18 @@ export async function* postUserMessage(
     };
     return;
   }
-  // Check plan limit
+  // Check plan and rate limit
   const isAboveMessageLimit = await isMessagesLimitReached({
     owner,
     plan,
-    mentions,
   });
   if (isAboveMessageLimit) {
     yield {
       type: "user_message_error",
       created: Date.now(),
       error: {
-        code: "plan_message_limit_exceeded",
-        message: "The message limit for this plan has been exceeded.",
+        code: "rate_limit_error",
```

Comment:
This is a user visible error. We likely want to have two messages based on what limit was reached. We can return a boolean and a type of limit exceeded and have 2 different messages here